src/command_modules/azure-cli-container/azure/cli/command_modules/container/_help.py

```diff
@@ -57,6 +57,11 @@
     short-summary: Examine the logs for a container in a container group.
 """
 
+helps['container exec'] = """
+    type: command
+    short-summary: Exec command for a container in a container group.
```

Comment:
"Execute command..."?  

Also, this is a non-trivial command and thus should have an example.  If this is only supported on Linux, then it should be called out in a long-summary.

Reply:
Also, from reviewing this PR, I don't think this help text is clear at all. I initially thought this command would execute a command on a specific container instance.  Now it seems like it basically opens a shell that tunnels into that container instance to run any number of commands.  Is this right?

Reply:
I agree.

Reply:
To address the second part. It will execute any command from within the container and stream the output back. The most common use case is to exec bash a the command which opens and interactive terminal from within the container. The example now shows doing this.